oeps/oep-0044-bp-changelogs.rst

```diff
@@ -0,0 +1,294 @@
+
+Changelogs and Changelog Entries
+--------------------------------
+
+Here are some recommendations for formatting changelogs and changelog entries, which have been taken from `keepachangelog.com`_.
+
+Guiding Principles
+~~~~~~~~~~~~~~~~~~
+
+* Changelogs are for humans, not machines.
+* There should be an entry for every single version [for libraries].
+* The same types of changes should be grouped.
+* Versions and sections should be linkable.
+* The latest version comes first.
+* The release date of each version is displayed.
+* Mention whether you follow Semantic Versioning.
+
+Types of Changes
+~~~~~~~~~~~~~~~~
+
+* **Added** for new features.
+* **Changed** for changes in existing functionality.
+* **Deprecated** for soon-to-be removed features.
+* **Removed** for now removed features.
+* **Fixed** for any bug fixes.
+
+These make great section headers, but similar language could also be used in messages to clarify the type of change.  For commits, you may want to use "adds" in place of "added", etc.
+
+The recommendation also includes "**Security** in case of vulnerabilities", but that is being called out separately because the Security Group needs to manage the timing of adding any such designation to any public changelog.
+
+Other Recommendations
+~~~~~~~~~~~~~~~~~~~~~
+
+* Format dates as YYYY-MM-DD. As an example, 2020-03-21.
+* If you do nothing else, list deprecations, removals, and any breaking changes in your changelog.
+* It is both acceptable and appropriate to clean up changelog entries at any time to add clarifications or customize messaging for the audience of the changelog when needed.
+
+Example Changelog
+~~~~~~~~~~~~~~~~~
+
+Here is an example CHANGELOG.rst.
+
+.. note:: Although this example includes versions, based on the decisions below our changelog entries would typically only include dates, because our versioned changelogs are managed using GitHub Releases.
+
+::
+
+  Changelog
+  ===========
+
+  All notable changes to this project will be documented in this file.
+
+  The format is based on recommendations in `OEP-44`_.
+  This project adheres to `Semantic Versioning`_.
+
+  .. _OEP-44: https://open-edx-proposals.readthedocs.io/en/latest/oep-0044-bp-changelogs.html
+  .. _Semantic Versioning: https://semver.org/spec/v2.0.0.html
+
+  [2.0.0] - 2020-03-21
+  --------------------
+
+  Removed
+  ~~~~~~~
+
+  * Removed deprecated function `xxx`. See DEPR-XXX for details.
+
+  [2.0.0] - 2020-03-21
+  --------------------
+
+  Added
+  ~~~~~
+
+  * Feature x. See announcement here: https://xxx
+
+Notable Differences
+~~~~~~~~~~~~~~~~~~~
+
+The following are notable differences between the recommendations of this OEP and the `keepachangelog.com`_ recommendations:
+
+* We use GitHub Releases in certain cases, as you'll see below, for developer efficiency purposes.
+* Because we typically release every change for libraries, we don't get any benefit from an "Unreleased" section.
```

Comment:
We don't seem to mention Unreleased except in this sentence. Perhaps we should make it explicitly optional, with reasons repo owners might choose to include it or not?

Reply:
I simply deleted this line.